sdks/java/container/build.gradle

```diff
@@ -22,6 +22,7 @@ plugins {
 }
 
 applyGoNature()
+applyPythonNature()
```

Comment:
Maybe add a comment that this is needed for `./license_scripts/license_script.sh`. Also, why is it that we need this now, but it seemed to work ok before?